lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart

```diff
@@ -26,103 +29,126 @@ class ExpansionContactListTile extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: const EdgeInsets.only(left: 8.0, top: 8.0, bottom: 12.0),
-      child: FutureBuilder<Profile?>(
-        key: contact.matrixId != null ? Key(contact.matrixId!) : null,
-        future:
-            contact.status == ContactStatus.active ? getProfile(context) : null,
-        builder: (context, snapshot) {
-          return Row(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              IgnorePointer(
-                child: Avatar(
-                  mxContent: snapshot.data?.avatarUrl,
-                  name: contact.displayName,
-                  size: 48.0,
+    return Container(
+      decoration: BoxDecoration(
+        border: Border(
+          bottom: BorderSide(
+            color: LinagoraStateLayer(LinagoraSysColors.material().surfaceTint)
+                .opacityLayer3,
+            width: 1,
+          ),
+        ),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.only(left: 8.0, top: 8.0, bottom: 8.0),
+        child: FutureBuilder<Profile?>(
+          key: contact.matrixId != null ? Key(contact.matrixId!) : null,
+          future: contact.status == ContactStatus.active
+              ? getProfile(context)
+              : null,
+          builder: (context, snapshot) {
+            return Row(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                IgnorePointer(
+                  child: Avatar(
+                    mxContent: snapshot.data?.avatarUrl,
+                    name: contact.displayName,
+                  ),
+                ),
+                const SizedBox(
+                  width: 12.0,
                 ),
```

Comment:
it should be 8

<img width="384" alt="image" src="https://github.com/user-attachments/assets/3c703b27-2c3c-44a2-8ab4-63cb04a7939e">

Reply:
This sizedBox is between the name and chips such owner or not-activated, its not displayed in the design, should i change it to 8 ?
![image](https://github.com/user-attachments/assets/06a76afc-0395-4628-803c-3b22315329d4)

Reply:
fixed